README.md

```diff
@@ -1 +1,19 @@
-# tic_tac_toe
\ No newline at end of file
+# Tic-Tac-Toe 
+
+## Introduction
+> This is a simple Tic-Tac-Toe game simply or poorly written by a beginner in Python.
+
+## How To Play
+> 1. Execute this Python file in the Command Prompt or run this in a Terminal.<br>
+> 2. Enter "Yes" to start the game.<br>
```

Comment:
Now no "yes" need to enter

Reply:
Updated! ce3e9ce7b71ffdb91d2948f314086da244902e1a